examples/canvas/app/boot.js

```diff
@@ -21,11 +21,11 @@ repo.addDomain('pixels', {
   advance (state, { x, y }) {
     return update(state, [x, y], this.rotateHue)
   },
-  register() {
+  register () {
     return {
-      [advance]: this.advance
+      [advance]: this.advance,
     }
-  }
+  },
```

Comment:
can trailing commas be configured? this stands out to me over most of the changes so far